lib/migration.js

```diff
@@ -186,7 +186,7 @@ function mixinMigration(MySQL, mysql) {
     function actualize(propName, oldSettings) {
       var newSettings = m.properties[propName];
       if (newSettings && changed(newSettings, oldSettings)) {
-        var pName = self.client.escapeId(propName);
+        var pName = self.columnEscaped(model, propName);
```

Comment:
```javascript
var pName = self.client.escapeId(newSettings.mysql && newSettings.mysql.columnName || propName);
```
do the same as
```javascript
var pName = self.columnEscaped(propName);
```


```javascript
/**
 * Get the escaped column name for a given model property
 * @param {String} model The model name
 * @param {String} property The property name
 * @returns {String} The escaped column name
 */
SQLConnector.prototype.columnEscaped = function(model, property) {
  return this.escapeName(this.column(model, property));
};
```
```javascript
/**
 * Get the column name for the given model property. The column name can be
 * customized at the model property definition level as `column` or
 * `columnName`. For example,
 *
 * ```json
 * "name": {
 *   "type": "string",
 *   "mysql": {
 *     "column": "NAME"
 *   }
 * }
 * ```
 *
 * @param {String} model The model name
 * @param {String} property The property name
 * @returns {String} The column name
 */
SQLConnector.prototype.column = function(model, property) {
  var prop = this.getPropertyDefinition(model, property);
  var columnName;
  if (prop && prop[this.name]) {
    columnName = prop[this.name].column || prop[this.name].columnName;
    if (columnName) {
      // Explicit column name, return as-is
      return columnName;
    }
  }
  columnName = property;
  if (typeof this.dbName === 'function') {
    columnName = this.dbName(columnName);
  }
  return columnName;
};
```